src/main/resources/schemas/primary_health_list.json

```diff
@@ -0,0 +1,311 @@
+            "required": [
+              "courtRoom",
+              "courtHouseName",
+              "courtHouseContact"
+            ],
+            "properties": {
+              "courtHouseName": {
+                "$id": "#root/courtLists/courtList/courtHouse/courtHouseName",
+                "description": "Name of the court house",
+                "title": "Court House Name",
+                "type": "string",
+                "default": "",
+                "examples": [
+                  "PRESTON"
+                ],
+                "pattern": "^.*$"
+              },
+              "courtHouseContact": {
+                "$id": "#root/courtLists/courtList/courtHouse/courtHouseContact",
+                "description": "The contact for the court house",
+                "title": "Court House Contact",
+                "$ref": "#/definitions/venueContact"
+              },
+              "courtHouseAddress": {
+                "$id": "#root/courtLists/courtList/courtHouse/courtHouseAddress",
+                "description": "The address for the court house",
+                "title": "Court House Address",
+                "$ref": "#/definitions/address"
+              },
+              "courtRoom": {
+                "$id": "#root/courtLists/courtRoom",
+                "type": "array",
+                "items": {
+                  "title": "Court Room",
+                  "description": "The court room that owns the court rooms",
+                  "type": "object",
+                  "required": [
+                    "session"
+                  ],
+                  "properties": {
+                    "session": {
+                      "$id": "#root/courtLists/courtRoom/session",
+                      "type": "array",
+                      "items": {
+                        "title": "Session",
+                        "description": "Session associated with the court in the publication",
+                        "type": "object",
+                        "required": [
+                          "sittings"
+                        ],
+                        "properties": {
+                          "sittings": {
+                            "$id": "#root/courtLists/courtList/courtRoom/session/sittings",
+                            "title": "Sittings",
+                            "type": "array",
+                            "default": [],
+                            "items": {
+                              "$id": "#root/courtLists/courtList/courtRoom/session/sittings/items",
+                              "type": "object",
+                              "required": [
+                                "sittingEnd",
+                                "hearing"
+                              ],
+                              "properties": {
+                                "sittingEnd": {
+                                  "$id": "#root/courtLists/courtList/courtRoom/session/sittings/items/sittingEnd",
```

Comment:
I think we'll need sittingStart here too, to account for 'Duration'

Reply:
Done :)